ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -152,6 +156,32 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-single-node-workers-upgrade-conformance
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported
```

Comment:
The reviewed profile should be `aws-ipi-shared-vpc-phz-f14`?

Reply:
^ doesn't support on 4.12 so we switched it to aws-ipi-byo-route53, see
https://redhat-internal.slack.com/archives/C065L72TA4U/p1701944156474429?thread_ts=1701269203.238639&cid=C065L72TA4U
I've updated the spreadsheet to reflect the change.